### PR TITLE
Update doc style

### LINKED ISSUE #180: Consistent documentation
Pass through all functions, and confirm that they all have consistent doc strings.

We use the [numpy documentation style guide](https://numpydoc.readthedocs.io/en/latest/format.html).

### PR DESCRIPTION
**Pull request by gwaygenomics**
# Description

Closes #180 

@niranjchandrasekaran - can you check this PR closely? By closely I also mean if I missed updating any functions.

## What is the nature of your change?

- [x] Bug fix (fixes an issue).
- [ ] Enhancement (adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] This change requires a documentation update.

# Checklist

Please ensure that all boxes are checked before indicating that a pull request is ready for review.

- [x] I have read the [CONTRIBUTING.md](CONTRIBUTING.md) guidelines.
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] I have deleted all non-relevant text in this pull request template.

### CODE REVIEW

File: pycytominer/cyto_utils/output.py [hunk 1]
```diff
@@ -96,8 +114,10 @@ def set_compression_method(compression):
 def check_compression_method(compression):
     """Ensure compression options are set properly
 
-    :param compression: the compression used to output data
-    :type compression: str
+    Parameters
+    ----------
+    compression : str
+        The category of compression options available
```
Comment (niranjchandrasekaran) [inline pycytominer/cyto_utils/output.py:120]: Missing Returns

File: pycytominer/cyto_utils/single_cell_ingest_utils.py [hunk 2]
```diff
@@ -26,8 +28,12 @@ def get_default_linking_cols():
...
     """Confirm that the linking cols and compartments are compatible
 
-    :return: Dictionary of compartment-specific column names used to link compartments across tables
-    :rtype: dict
+    Parameters
+    ----------
+    linking_cols : str or dict, default "default"
+        Specify how to link objects
+    compartments : str or list, default "default"
+        Which compartments used in the experiment.
```
Comment (niranjchandrasekaran) [inline pycytominer/cyto_utils/single_cell_ingest_utils.py:36]: Missing Returns

File: pycytominer/operations/transform.py [hunk 3]
```diff
@@ -102,28 +123,42 @@ def fit(self, X, y=None):
         return self
 
     def transform(self, X, y=None):
-        """
-        Perform the sphering transform
+        """Perform the sphering transform
```
Comment (niranjchandrasekaran) [inline pycytominer/operations/transform.py:126]: Missing Parameters

**Review by niranjchandrasekaran — APPROVED** [review]
@gwaygenomics I found a few `Returns` missing in the docstrings. I have highlighted those. Apart from that, as you probably know already, some functions have helpful examples in the docstring while others don't. Perhaps we should make it a task for version 0.2.

Lastly, in `pycytominer/cyto_utils/load.py` docstrings should say `Parameters` instead of `Arguments` and `Returns` instead of `Return`.  Also `load_npz()` is missing `Returns`. 

Everything else looks good.

**Comment by gwaygenomics** [comment]
Thanks for the careful review @niranjchandrasekaran - I've addressed all your comments, and will now merge!

**Pull request merged**